Cactooz/StarWarsWiki
# Friends profile styling

### Changes
- Add styling to the profile page.
- Different colored buttons for X and Check.
- Changed the text that is showed on other peoples profiles.
- Add titles to all buttons/links so a popup is shown on hover.
- Fixed so the favorites title is only shown if the profile has any favorites.

### What to test

- Check others profiles with and without favorites.
- Try to add, get and accept friend requests.
- Remove all your favorites and check how it looks.
